### DISCUSSION TITLE
When I am loading collection, a message returned No online queryNode to allocate

### DISCUSSION THREAD

**Discussion by AliscaCL** [milvus-io/milvus#15812] [Q&A and General discussion] [answered]
When I am loading collection, a message returned No online queryNode to allocate. 
What happened?

**Comment by xiaofan-luan** [comment, marked as answer]
please check if you have enough memory to load the collection

**Comment by AliscaCL** [comment]
Sorry for disturbing.... It seems, my server is down...
